enhancements/console/customize-perspectives.md

```diff
@@ -0,0 +1,278 @@
+---
+title: customize-perspectives
+authors:
+  - "@lokanandaprabhu"
+reviewers:
+  - "@jerolimov"
+  - "@spadgett"
+approvers:
+  - "@spadgett"
+  - "@cvogt"
+api-approvers:
+  - "@jerolimov"
+  - "@cvogt"
+creation-date: 2022-07-26
+last-updated: 2022-08-24
+tracking-link: 
+  - https://issues.redhat.com/browse/ODC-6695
+status: implementable
+---
+
+# Customize Perspectives 
+
+## Summary
+
+This document provides details about how cluster administrator 
```

Comment:
It would be good to know what a user perspective is.

> In the OpenShift Developer Console, Perspectives are ...

Reply:
Added the screenshot